Example/StripeConnectExample/StripeConnectExample/AppSettingsView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ServerSettingsView.swift
+//  StripeConnect Example
+//
+//  Created by Chris Mays on 8/26/24.
+//
+
+import SwiftUI
+
+struct AppSettingsView: View {
+    
+    @Environment(\.dismiss) var dismiss
+    @Environment(\.viewControllerPresenter) var viewControllerPresenter
+    
+    // App info is sometimes nil, if for example it fails to load.
+    var appInfo: AppInfo?
+    
+    @State var selectedMerchant: MerchantInfo?
+    @State var serverURLString: String = AppSettings.shared.selectedServerBaseURL
+    
+    var isCustomEndpointValid: Bool {
+        guard let url = URL(string: serverURLString) else { return false }
+        return UIApplication.shared.canOpenURL(url)
+    }
+    
+    var saveEnabled: Bool {
+        isCustomEndpointValid &&
+        (AppSettings.shared.selectedMerchant(appInfo: appInfo)?.id != selectedMerchant?.id ||
+         AppSettings.shared.selectedServerBaseURL != serverURLString)
+    }
+    
+    init(appInfo: AppInfo?) {
+        self.appInfo = appInfo
+        _selectedMerchant = .init(initialValue: AppSettings.shared.selectedMerchant(appInfo: appInfo))
+    }
+    
+    var body: some View {
+        NavigationView {
+            List {
+                Section {
+                    ForEach((appInfo?.availableMerchants ?? [])) { merchant in
+                        OptionListRow(title: merchant.displayName ?? "",
+                                      subtitle: merchant.id,
+                                      selected: selectedMerchant?.id == merchant.id,
+                                      onSelected: {
+                            selectedMerchant = merchant
+                        })
+                    }
+                } header: {
+                    Text("Select a demo account")
+                }
+                Section {
+                    TextInput(label: "", placeholder: "https://example.com", text: $serverURLString, isValid: isCustomEndpointValid)
```

Comment:
In addition to specifying a custom server, it's actually more important that we be able to specify additional accounts, beyond just the canonical ones hardcoded on the backend. This will enable us to quickly create test accounts and test them agains the example app during team bug bashes.

But I think you can address this in a separate PR – this work was spec'd in a separate ticket (MXMOBILE-2481) anyway and you ended up including most of it here 😄